rules/linux/privilege_escalation_awk_binary_shell.toml

```
@@ -0,0 +1,76 @@
+[metadata]
+creation_date = "2022/02/24"
+maturity = "production"
+updated_date = "2022/02/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Linux binary mawk or nawk abuse to breakout out of restricted shells or environments by spawning an
+interactive system shell.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux Restricted Shell Breakout via nawk /mawk Shell Evasion"
```

There are 4 binaries reference in the query. Maybe can change this to 
```suggestion
name = "Linux Restricted Shell Breakout via awk Commands"
```